src/routes/github/create-branch/github-create-branch-post.ts

```diff
@@ -40,6 +40,20 @@ export const GithubCreateBranchPost = async (req: Request, res: Response): Promi
 		res.sendStatus(200);
 	} catch (err) {
 		req.log.error({ err }, "Error creating branch");
-		res.sendStatus(500);
+		res.status(500).json(verifyError(err));
 	}
 };
+
+const verifyError = (error) => {
+	switch (error.status) {
```

Comment:
/nit
/Opinion piece

Thoughts on Object map instead of switch? Could replace the `verifyError` function.
i've ignored the fact the messaged are an array, but as an example......

```
const errorMessages = {
    401: "This GitHub repo.."
    403: "This GitHub repo..",
    default: "default message..."
}

```
`res.status(500).json(verifyError(err));` can be 
`res.status(500).json(errorMessages[err?.status] || errorMessages.default);`

Reply:
Updated